HoneCtrl.bat

```diff
@@ -3161,8 +3161,11 @@ goto HoneRenders
 :RenderSettings
 cls
 if not exist "%SYSTEMDRIVE%\Program Files\VEGAS\VEGAS Pro 17.0" goto NoVegas
+if not exist "%SYSTEMDRIVE%\Program Files\VEGAS\VEGAS Pro 17" goto NoVegas

```

Comment:
Starting from line 2, it will always goto NoVega, to reach Taskkill all these folders should exist lol, since we're not using many if else statements

Reply:
I believe that I have changed it.

Reply:
Before actually tweaking, it goes through like 5 If not exist, do you not see the problem. Should add if not exist XXX else goto RenderSettings2 or something